grpc/src/main/java/io/stargate/grpc/service/SingleExceptionHandler.java

```diff
@@ -0,0 +1,29 @@
+package io.stargate.grpc.service;
+
+import io.grpc.Metadata;
+import io.grpc.Status;
+import io.grpc.stub.StreamObserver;
+import io.stargate.proto.QueryOuterClass;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class SingleExceptionHandler extends ExceptionHandler {
```

Comment:
should we call this better `TerminatingExceptionHandler` or smth like this?

Reply:
I would prefer to have this distinction as `Single` and `Streaming`

Reply:
OK, as you wish..